src/vuestic-theme/vuestic-components/va-dropdown/VaDropdown.vue

```diff
@@ -0,0 +1,298 @@
+<template>
+  <div>
+    <slot name="actuator">
+      <div>Actuator</div>
+    </slot>
+    <transition :name="noFade ? '' : 'va-dropdown__transition'" mode="out-in">
+      <div
+        v-show="isVisible"
+        ref="dropdown"
+        :style="dropdownStyles"
+        class="va-dropdown"
+        :class="className"
+      >
+        <slot>{{ message }}</slot>
+      </div>
+    </transition>
+  </div>
+</template>
+
+<script>
+import _ from 'lodash'
+const availablePositions = ['T', 'TR', 'R', 'BR', 'B', 'BL', 'L', 'TL']
+const availableTriggerModes = ['hover', 'click', 'focus']
+
+export default {
+  name: 'va-dropdown',
+  props: {
+    position: {
+      type: String,
+      default: 'T',
+      validator: position => availablePositions.indexOf(position) >= 0
+    },
+    triggerMode: {
+      type: String,
+      default: 'click',
+      validator: triggerMode => availableTriggerModes.indexOf(triggerMode) >= 0
+    },
+    disabled: {
+      type: Boolean,
+      default: false
+    },
+    maxWidth: {
+      type: [String, Number]
+    },
+    maxHeight: {
+      type: [String, Number]
+    },
+    noFade: Boolean,
+    message: String,
```

Comment:
What's the idea here? I think I should be able to pass in anything I want via slot, no?